### PR TITLE
Changing cursor to pointer when hover on track

### PR DESCRIPTION
**Pull request by meet503186**
## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by meet503186** [comment]
@googlebot I signed it!

**Comment by meet503186** [comment]
@googlebot I signed it!

**Comment by meet503186** [comment]
@joeyparrish please review this change

File: ui/less/range_elements.less [hunk 1]
```diff
@@ -57,6 +57,11 @@
   /* The track should fill the range element. */
   width: 100%;
 
+ /* Since range elements are special input elements so they should reflect,
+   * user interaction so whhen user hover on range element the cusror should
+   * be pointer. */
+  cursor: pointer;
+
```
Comment (aditmehta9) [inline ui/less/range_elements.less:64]: Hello @meet503186 this is already implemented in [PR](https://github.com/google/shaka-player/pull/3220/files)

Comment (meet503186) [reply ui/less/range_elements.less:64]: Ok then i close this PR

**Pull request closed without merging**